conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conan import ConanFile
+from conan.tools.cmake import cmake_layout, CMakeToolchain
+from conan.tools.system.package_manager import Apt
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=2.0.0 <3"
+
+
+class BSMPT(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps"
+
+    options = {
+        "enable_tests": [True, False],  # enables the unit tests
```

Comment:
I propose to rename this to `EnableTests` to be consistent with the other names.